code/modules/religion/sects/shadow_sect.dm

```diff
@@ -60,6 +64,88 @@
 	icon_state = "shadow-obelisk"
 	anchored = FALSE
 	break_message = "<span class='warning'>The Obelisk crumbles before you!</span>"
+	var/heal_delay = 20
+	var/last_heal = 0
+	var/spread_delay = 45
+	var/last_spread = 0
+
+/obj/structure/destructible/religion/shadow_obelisk/Initialize(mapload)
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	. = ..()
+	if(!sect.faithful_used)
+		return
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/destructible/religion/shadow_obelisk/LateInitialize()
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	. = ..()
+	if(!sect.faithful_used)
+		return
+	START_PROCESSING(SSobj, src)
+
+/obj/structure/destructible/religion/shadow_obelisk/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/structure/destructible/religion/shadow_obelisk/process(delta_time)
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	if(sect.faithful_used)
```

Comment:
Use an early return if there is no alternate behaviour

Reply:
so, 
```
if(sect.faithful_used)
	stuff here
else
	return
```
?

Reply:
wait I think I got it

Reply:
```
	if(!sect.faithful_used)
		return
```